### ISSUE TITLE
Fix contrast of the header Extensions/Themes links

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#3253]
The contrast of the Extensions/Themes links grey on a dark background is not that great.

<img width="422" alt="colour_contrast_analyser_and_new_issue_ _mozilla_addons-frontend" src="https://user-images.githubusercontent.com/1514/30908714-5c1800e2-a376-11e7-9333-edbfcc01287a.png">

If we update to the same grey used in the footer links (which is not a photon colour) we get:

<img width="428" alt="colour_contrast_analyser_and_new_issue_ _mozilla_addons-frontend" src="https://user-images.githubusercontent.com/1514/30908756-92090c28-a376-11e7-8fbd-d91302542050.png">

Before:
<img width="1401" alt="add-ons_for_firefox" src="https://user-images.githubusercontent.com/1514/30908826-c1337d26-a376-11e7-9d09-e9ac88cf7437.png">

After:
<img width="1428" alt="add-ons_for_firefox" src="https://user-images.githubusercontent.com/1514/30908793-aee3b0f0-a376-11e7-8f2d-f75deba6be03.png">

**Comment by tofumatt** [comment]
I think we should change it regardless because failing accessibility is bad but is there a photon colour that’s close we could use instead?

Sad the footer isn’t photon at all though.

**Comment by muffinresearch** [comment]
> Sad the footer isn’t photon at all though.

Yeah another alternative is to update all of these greys to the same photon grey. That would be an easy change too.

**Comment by muffinresearch** [comment]
$grey-40 is the closest:

<img width="447" alt="colour_contrast_analyser_and_color_ _visuals___photon_design_system" src="https://user-images.githubusercontent.com/1514/30912933-475d65e2-a386-11e7-80ae-50cbbed5d05f.png">

**Comment by tofumatt** [comment]
That works for me. 👍🏻

**Comment by pwalm** [comment]
Grey 40 it is! I believe that grey I originally used it a Photon colour, but it might have been before the expanded the grey palette.

**Comment by ValentinaPC** [comment]
Verified as fixed on AMO-dev FF55(Win 7).
Postfix video:
![contrast](https://user-images.githubusercontent.com/15685960/30953557-21c63494-a435-11e7-9d92-dd503620b206.gif)
